inc/includes.php

```diff
@@ -105,15 +98,14 @@
    $PLUGINS_INCLUDED = 1;
    $LOADED_PLUGINS   = [];
    $plugin           = new Plugin();
-   if (!isset($_SESSION["glpi_plugins"])) {
+   if ($plugin->getPlugins() === null) {
```

Comment:
`if (count($plugin->getPlugins()) === 0) {`

Reply:
Hum... We're originally trying to knw here if "something" has been loaded; we definitely cannot rely on the `getPlugins` method.